LEGO1/lego/legoomni/include/legocarbuildpresenter.h

```diff
@@ -4,7 +4,15 @@
 #include "anim/legoanim.h"
 #include "legoanimpresenter.h"
 
+// SIZE 0xc
```

Comment:
Should be `0x0c`. Also maybe this struct should be defined within LegoCarBuildAnimPresenter itself?